Sources/GDSCommon/Components/ContentTile/ContentTileView.swift

```diff
@@ -0,0 +1,207 @@
+import Foundation
+import UIKit
+
+public final class ContentTileView: NibView {
+    public let viewModel: ContentTileViewModel
+    
+    public init(frame: CGRect, viewModel: ContentTileViewModel) {
+        self.viewModel = viewModel
+        super.init(frame: frame, bundle: .module)
+    }
+    
+    required public init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @IBOutlet private var containerView: UIView! {
+        didSet {
+            containerView.layer.cornerRadius = 16
+            containerView.layer.masksToBounds = true
+            containerView.accessibilityIdentifier = "containerView"
+        }
+    }
+    
+    @IBOutlet private var containerStackView: UIStackView! {
+        didSet {
+            containerStackView.accessibilityIdentifier = "containerStackView"
+            
+            containerStackView.addSubview(closeButton)
+            NSLayoutConstraint.activate([
+                closeButton.trailingAnchor.constraint(greaterThanOrEqualTo: containerStackView.trailingAnchor, constant: -16),
+                closeButton.topAnchor.constraint(greaterThanOrEqualTo: containerStackView.topAnchor, constant: 8)
+            ])
+        }
+    }
+    
+    @IBOutlet private var imageView: UIImageView! {
+        didSet {
+            imageView.accessibilityIdentifier = "content-tile-image"
+            if let view = viewModel as? ContentTileViewModelWithImage {
+                guard view.image.size.height > 0 else {
```

Comment:
I would be tempted to merge this with the `if` statement above so that we fall down the else path and hide the image view if the image is empty 🙂

Reply:
Hmm.. maybe I'm doing it wrong, but I'm not able to merge them without creating a different variable (i.e keeping the `let view = ...`)